ethexe/network/src/lib.rs

```diff
@@ -416,7 +416,18 @@ impl NetworkService {
             .gossipsub
             .publish(gpu_commitments_topic(), data)
         {
-            log::debug!("gossipsub publishing failed: {e}")
+            log::error!("gossipsub publishing failed: {e}")
+        }
+    }
+
+    pub fn publish_transaction(&mut self, data: Vec<u8>) {
```

Comment:
offchain_transaction. I'd go with this naming everywhere in case for future features append etc

Reply:
done